packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -0,0 +1,53 @@
+Name:     foreman-release
+Version:  1.19.0
+Release:  1%{?dist}
```

Comment:
This needs a develop to differentiate between nightly "1.19" and actual 1.19 (which has GPG checking).